src/RageSoundPosMap.cpp

```diff
@@ -136,16 +122,22 @@ std::int64_t pos_map_queue::Search( std::int64_t iSourceFrame, bool *bApproximat
 	/* iSourceFrame is probably in pos_map.  Search to figure out what position
 	 * it maps to. */
 	std::int64_t iClosestPosition = 0, iClosestPositionDist = INT_MAX;
-	const pos_map_t *pClosestBlock = &*m_pImpl->m_Queue.begin(); /* print only */
+	
+	/* pClosestBlock is ONLY used for the "Approximate sound time:" log trace.
```

Comment:
You can remove all these commented out pieces of code (and this comment as well). Here and below.